content/posts/pandoc/pandoc.md

```diff
@@ -0,0 +1,334 @@
+---
+title: "How I escaped from Microsoft Office tools as a Computer Science student"
+date: 2019-10-09T00:35:20+07:00
+draft: true
+author: "kn"
+tags: ["pandoc", "presentation", "blog"]
+cover: "/posts/pandoc/images/the-powerpoint-was.jpg"
+keywords: ["pandoc"]
+description: "The Easiest Way to Make Presentations! (Pandoc + Markdown)"
+showFullContent: false
+---
+
+## Introduction and Motivation
+![](/posts/pandoc/images/mdVSw.png)
+
+I have used beamer quite a lot to prepare slides for both research and class presentations. I have started to use LaTeX to content in beamer presentation for two years. But for some time I've been concerned about the high ratio of markup to content in beamer presentations.  used to type thousands lines of code just to create a presentation slide which was tiring and time consuming. Thus, I did some research and found a combination of pandoc, markdown, and LaTeX to simplify my process. In my point of view, markdown in syntax is straightforward, easy to read and modify. Moreover, pandoc has the ability to convert markdown files into LaTeX, hence enabled me to customise my slides greatly later on.
```

Comment:
- I *have been using* LaTex ...
- ~~But for~~ *sometimes* I *was concerned* ...
- In *my opinion*, ...
- hence *enables* ...